.travis.yml

```diff
@@ -2,21 +2,16 @@ language: node_js
 sudo: false
 dist: precise
 node_js:
+    - "node"
```

Comment:
is this how we specify node 8? Would prefer actually putting "8" if thats allowed or something that indicates that we are testing v8.